messaging/src/main/java/org/phantazm/messaging/packet/c2s/RoundStartPacket.java

```diff
@@ -0,0 +1,28 @@
+package org.phantazm.messaging.packet.c2s;
+
+import org.jetbrains.annotations.NotNull;
+import org.phantazm.messaging.packet.Packet;
+import org.phantazm.messaging.serialization.DataReader;
+import org.phantazm.messaging.serialization.DataWriter;
+
+import java.util.Objects;
+
+public record RoundStartPacket() implements Packet {
+
+    public static final byte ID = 0;
+
+    public static @NotNull RoundStartPacket read(@NotNull DataReader reader) {
+        Objects.requireNonNull(reader, "reader");
+        return new RoundStartPacket();
+    }
+
+    @Override
+    public byte getId() {
+        return ID;
```

Comment:
same ID as MapDataVersionQueryPacket

Reply:
packets are grouped by channel
(also I might get rid of packet IDs bc I see that minecraft prefers using custom payload packets with one channel for each packet)